### ISSUE TITLE
Sender pod cannot create mirror interface

### ISSUE THREAD

**Issue by leathersole** [redhat-nfvpe/kokotap#12]
Hi,

Sender pod cannot create `mirror` interface, and it output this log:
```
sender
Waiting for signal at main ...
XXX:failed to get mirror MTU: failed to lookup "mirror": Link not found
```
There was 2 lines that contains `mirror` in the system log.
```
Mar 28 11:55:09 n2.example.com NetworkManager[3343]: <warn>  [1553741709.1196] device (mirror): could not get vxlan properties
Mar 28 11:55:09 n2.example.com NetworkManager[3343]: <info>  [1553741709.1200] manager: (mirror): new Vxlan device (/org/freedesktop/NetworkManager/Devices/57)
```

Environment:
- RHEL 7.6 on libvirt
- OpenShift Container Platform 3.11
- kokotap 0.6
```
oc get node
NAME                        STATUS    ROLES          AGE       VERSION
n0.example.com   Ready     infra,master   26d       v1.11.0+d4cacc0
n1.example.com   Ready     compute        26d       v1.11.0+d4cacc0
n2.example.com   Ready     compute        26d       v1.11.0+d4cacc0
```

Executed command:
```
./kokotap --pod=system-app-1-f4v87 --mirrortype=both --dest-node=n1.example.com --vxlan-id=100 --kubeconfig=$HOME/.kube/config --namespace=3scale24 --vxlan-port=4889|oc create -f -
```
`system-app-1-f4v87` is on n2.example.com.

Just FYI: Receiver pod created `mirror` interface:
```
receiver
Waiting for signal at main ...
```
```
46: mirror: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1450 qdisc noqueue state UNKNOWN group default qlen 1000
    link/ether ea:76:10:e5:3f:7c brd ff:ff:ff:ff:ff:ff
    inet6 fe80::e876:10ff:fee5:3f7c/64 scope link 
       valid_lft forever preferred_lft forever
```

**Comment by s1061123** [comment]
Hi, thank you for the issue!
Fixed it in 4574ac82c06d496a052bdd0caece3b3c9213f340, so please use v0.61 (fixed version).

BTW, mirror interface of sender, cannot be seen from host because mirror interface is in pod namespace.
Please use `nsenter -t <pid> -n -- ip address show` to see the interface.

**Comment by leathersole** [comment]
Thanks Tomofumi! kokotap v0.61 is now working on my cluster!
